mikeio1d/result_network/result_location.py

```diff
@@ -65,6 +70,22 @@ def _repr_html_(self) -> str:
         )
         return repr
 
+    def read(self, column_mode: Optional[str | ColumnMode] = None) -> pd.DataFrame:
```

Comment:
Read the time series data into a data frame of all quantities for a particular location?